packages/main/src/plugin/docker-extension/docker-desktop-installation.ts

```diff
@@ -176,8 +186,10 @@ export class DockerDesktopInstallation {
     return builtError;
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  ipcHandle(channel: string, listener: (event: IpcMainInvokeEvent, ...args: any[]) => Promise<void> | any): any {
+  ipcHandle(
+    channel: string,
+    listener: (event: IpcMainInvokeEvent, ...args: string[]) => Promise<void> | unknown,
```

Comment:
Looks a bit weird to have `Promise<void> | unknown`, some listener might not have a proper return type?

Reply:
removed unknown